demos/menu-surface.html

```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2018 Google Inc. All rights reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+-->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Menu Surface - Material Components Catalog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" type="image/png" href="/images/logo_components_color_2x_web_48dp.png">
+    <script src="/ready.js"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto+Mono">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="/assets/menu-surface.css">
+    <style>
```

Comment:
Isn't this where the demo specific styles go unless we're showing something ?

Reply:
There should be no `style` tag ideally. Everything should be in the `scss` so we can use mixins directly.

Reply:
Nearly every demo has a `style`tag with layout specific styling if we're not showing how to do something in a sass file. We can change this, but we should also create an issue so that we're being consistent.

Reply:
Yeah, only a few demo pages were created after the point where we had individual SCSS files for each demo page, and probably even fewer pre-existing demo pages were fully converted to put all their styles in the SCSS file. Card is an example though.

We're not going to spend additional time on demo page consistency at this point, because the catalog supersedes the demo pages, and the demo pages should be retired once we have screenshot test pages for everything.